src/DistributedComputations/distributed_architectures.jl

```diff
@@ -83,18 +83,29 @@ specifically defined by `Int`, `Fractional`, or `Sizes`.
 """
 struct Equal end
 
-"""type representing fractional domain partioning where worker `i` is allocates `sizes[i] / sum(sizes)` parts of the domain"""
 struct Fractional{S} 
     sizes :: S
 end
 
-"""type representing domain partioning where worker `i` holds `sizes[i]` parts of the domain"""
 struct Sizes{S} 
     sizes :: S
 end
 
+"""
+    Fractional(ϵ1, ϵ2, ..., ϵN)
+
+Return a type that partitions a direction unequally. The total work is `W = sum(ϵi)`, 
+and each process is then allocated `ϵi / W` of the domain.
+"""
 Fractional(args...) = Fractional(tuple(args ./ sum(args)...))  # We need to make sure that `sum(R) == 1`
-     Sizes(args...) = Sizes(tuple(args...))
+
+"""
+`Sizes(ϵ1, ϵ2, ..., ϵN)`
```

Comment:
I used `ϵ` for fractions because that made sense but we should use a different letter for sizes.

It's good to get this right because users will copy the notation we use in docstrings, so it propagates everywhere.